src/server/kernel/wlayersurface.h

```diff
@@ -111,6 +112,7 @@ class WAYLIB_SERVER_EXPORT WLayerSurface : public WToplevelSurface, public WObje
     void bottomMarginChanged();
     void exclusiveZoneChanged();
     void keyboardInteractivityChanged();
+    void requestCommit();
```

Comment:
How to use it?

Reply:
We should not monitor the changes of `one single PROPERTY`, should handle them together when commit.

Can avoid sending configure to client when some properties have been processed, while others have not

Reply:
I‘d like to rename `requestPropertyChanged`

Reply:
`layerPropertiesChanged` is better.